crates/testing/src/helpers.rs

```diff
@@ -291,16 +291,12 @@ pub fn build_payload_commitment<TYPES: NodeType>(
 }
 
 /// TODO: <https://github.com/EspressoSystems/HotShot/issues/2821>
-#[allow(clippy::type_complexity)]
 pub fn build_vid_proposal<TYPES: NodeType>(
     quorum_membership: &<TYPES as NodeType>::Membership,
     view_number: TYPES::Time,
     transactions: Vec<TestTransaction>,
     private_key: &<TYPES::SignatureKey as SignatureKey>::PrivateKey,
-) -> (
-    Proposal<TYPES, VidDisperse<TYPES>>,
-    Vec<Proposal<TYPES, VidDisperseShare<TYPES>>>,
-) {
+) -> VidProposal<TYPES> {
```

Comment:
This name could be different. Just based on the function it seemed like an obvious name.